lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -422,6 +422,17 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Path to <b>newrelic.yml</b>. If undefined, the agent checks the following directories (in order): <b>config/newrelic.yml</b>, <b>newrelic.yml</b>, <b>$HOME/.newrelic/newrelic.yml</b> and <b>$HOME/newrelic.yml</b>.'
         },
+        :'custom_events_instrumentation_topics' => {
+          :default => [],
+          :public => true,
+          :type => Array,
+          :allowed_from_server => true,
+          :dynamic_name => true,
```

Comment:
I don't think having this marked as a dynamic name is necessary since you are calling this config directly in `lib/new_relic/agent/instrumentation/rails_notifications/custom_events.rb:29`. It should be safe to just remove this line.

Reply:
Updated with https://github.com/newrelic/newrelic-ruby-agent/pull/1659/commits/951476b03bd2beafa3f9804a04b79b0b8a88ba6f